### PR TITLE
Fixed Shelter Profile Adoption Bug

### PR DESCRIPTION
**Pull request by gracebarsny**
Fixed bug where you couldn't adopt a dog, click next on the dog shelter profile, and adopt the next one.

**Scripts**
- DogProfile: removed unused libraries, added dog variable, added protected accessor for dog's info, cleaned up styles, set dog variable in SetProfile
- DogShelterProfile: removed dog variable (in superclass now)

### CODE REVIEW

File: Assets/Scripts/UI/PickupPup/DogProfile/DogProfile.cs [hunk 1]
```diff
@@ -85,14 +91,14 @@ void Update()
...
-        dogInfo = dog.Info;
+        this.dog = dog;
 		buttonController.CalibrateIndex (dog);
 
         nameText.text = dogInfo.Name;
         breedText.text = dogInfo.Breed.Breed;
-		if (descriptionText != null) {
-			for (int i = 0; i < descriptionText.Length; i++) {
-				if (i < dogInfo.Descriptions.Length) {
+		if(descriptionText != null) {
```
Comment (imann24) [inline Assets/Scripts/UI/PickupPup/DogProfile/DogProfile.cs]: Edit for bracket conventions

File: Assets/Scripts/UI/PickupPup/DogProfile/DogProfile.cs [hunk 2]
```diff
@@ -85,14 +91,14 @@ void Update()
...
+			for(int i = 0; i < descriptionText.Length; i++) {
+				if(i < dogInfo.Descriptions.Length) {
 					descriptionFields [i].Show ();
```
Comment (imann24) [inline Assets/Scripts/UI/PickupPup/DogProfile/DogProfile.cs]: This is old, but also has an extra blank space

**Comment by imann24** [comment]
@gracebarsny did you note the revision comments on here?

**Comment by gracebarsny** [comment]
@imann24 I see them now!

**Pull request merged**